### ISSUE TITLE
Postnet does not update at all in FastSpeech2

### ISSUE THREAD

**Issue by iamanigeeit** [espnet/espnet#4684]
While pruning FastSpeech2 using [connection gradient](https://arxiv.org/abs/1810.02340), i realized that the entire postnet is being pruned with gradient 0.

Then i checked my FS2 checkpoints and found that the postnet indeed does not get updated at all. 
```
import torch
m1 = torch.load('/home/perry/PycharmProjects/espnet/egs2/ljspeech/tts1/exp/test/1epoch.pth')
m2 = torch.load('/home/perry/PycharmProjects/espnet/egs2/ljspeech/tts1/exp/test/2epoch.pth')
for name, param in m1.items():
    if 'postnet' in name:
        if 'weight' in name or 'bias' in name:
            print(name, param.equal(m2[name]))
                                
tts.postnet.postnet.0.0.weight True
tts.postnet.postnet.0.1.weight True
tts.postnet.postnet.0.1.bias True
tts.postnet.postnet.1.0.weight True
tts.postnet.postnet.1.1.weight True
tts.postnet.postnet.1.1.bias True
tts.postnet.postnet.2.0.weight True
tts.postnet.postnet.2.1.weight True
tts.postnet.postnet.2.1.bias True
tts.postnet.postnet.3.0.weight True
tts.postnet.postnet.3.1.weight True
tts.postnet.postnet.3.1.bias True
tts.postnet.postnet.4.0.weight True
tts.postnet.postnet.4.1.weight True
tts.postnet.postnet.4.1.bias False
```

What is the role of the postnet?

**Comment by kan-bayashi** [comment]
Maybe this is related to the issue https://github.com/espnet/espnet/issues/4555.

**Comment by iamanigeeit** [comment]
I confirm that this is related to the initialization issue. Thanks!
